lib/cmake/scipp-conan.cmake

```diff
@@ -50,7 +50,7 @@ if(SKBUILD)
     LLNL-Units:shared=False
     LLNL-Units:fPIC=True
     LLNL-Units:base_type=uint64_t
-    LLNL-Units:namespace=llnl::units
+    LLNL-Units:namespace=units
```

Comment:
Wait you set it here explicitly, but at `llnl-units/conanfile.py` you don't seem to process this option? I'd remove this line here and rely on upstream to maintain the same default namespace.